.github/workflows/test.yml

```diff
@@ -106,9 +106,9 @@ jobs:
       # TODO(https://pathcheck.atlassian.net/browse/SAF-250): remove the skip on COVIDSafePathsTests/testRendersWelcomeScreen once it's passing
       - run: |
           xcodebuild test -workspace COVIDSafePaths.xcworkspace \
-            -scheme COVIDSafePathsTests \
+            -scheme GPS_Tests \
```

Comment:
Ended up needing to split the test targets as well. COVIDSafePathsTests became GPSTests